manticore/platforms/evm.py

```diff
@@ -1049,6 +1051,41 @@ def __reduce__(self):
         return (self.__class__, (self.data, ))
 
 
+def concretized_args(**policies):
+    '''
+    Make sure an EVM instruction has all of its arguments concretized according to
+    provided policies.
+
+    Example decoration:
+
+        @concretized_args(size='ONE')
+        def LOG(self, address, size, *topics):
+            ...
+
+    The above will make sure that the |size| parameter to LOG is Concretized when symbolic
+    according to the 'ONE' policy.
+
+    :param policies: A kwargs list of argument names and their respective policies.
+                         Provide None or '' as policy to use default.
```

Comment:
I'd rather require just one value to make it default. `None`?

Then it could be `policy = policy if policy is not None else 'MINMAX'`

Reply:
I thought briefly about this, and I ended up defaulting it if falsifiable values are provided too to make the call site slightly easier. (i.e. `size=''` is more succinct than `size=None`).